src/lib/merkle_ledger/util.ml

```diff
@@ -99,17 +101,19 @@ end = struct
       (compute_affected_locations_and_hashes t locations_and_hashes
          locations_and_hashes)
 
+  (* When we do batch on a database, we should add accounts and hashes
```

Comment:
At the beginning of the comment, add TODO. It makes it easy to find that we are going to make this change in the future.